package.json

```diff
@@ -11,23 +11,7 @@
     "changed": "yarn lerna changed",
     "release": "yarn lerna publish",
     "bootstrap": "yarn lerna bootstrap",
-    "build": "run-s build:types build:shared build:base build:db build:sdk build:closure",
```

Comment:
I think we had these separate commands because in the monorepo you don't necessarily need to rebuild all dependencies all the time or have we got this wrong?

Reply:
Your monorepo acts as a single unit. Since (now) everything builds/flows thru the same pipeline, you should keep that mindset and apply it the rest.

Individual packages should only contain custom differences, much like how packages' `tsconfig.json` files extend a base, and then modify/add individual settings.

An example of this would be `firemock` and `docs` packages including the commands (and extra dev deps) needed to generate the documentation sites.